packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -20,6 +21,19 @@ class NodeCreate(InputSchemaWithoutCamelCase):
     service_id: str | None
 
 
+class BootOptions(BaseModel):
+    boot_mode: str
```

Comment:
The `bootOptions` is actually defined on the service and its specific cor each different service.
It is an unfortunate coincidence that we use use always the same value. We can have multiple entries for this and boot_mode is not mandatory or enforced at any point.


Please define it as `Any` with a note that hints this value is service specific and boot_mode is not the only option there, or could be a different name.

```suggestion
BootOptions: TypeAlias = Any
```

Reply:
Just wanted to double-check with you because, from the DB entries, it seems that the only values present are {"bootOptions": {"boot_mode": "1"}} with values 1, 2, or 3. Therefore, are you sure that these boot options refer to what you are talking about, or are they just something the frontend uses?